packages/frontend/src/components/topicsMenu.tsx

```diff
@@ -0,0 +1,39 @@
+import { useHistory } from 'react-router-dom'
+
+const TopicsMenu = () => {
+    const history = useHistory()
+
+    const goToPhil = () => {
+        history.push('/philosophy', { isConfirmed: true })
+    }
```

Comment:
I think it can be like
```tsx
const goToTopic = () => {
    history.push(`/${topic}`, { isConfirmed: true })
}
```

Reply:
Where are you getting the `topic` value from?

Reply:
Ah, nevermind I think I can do something like this:

 ```ts
const goToTopic = (topicId) => {
    history.push(`/${topicId}`, { isConfirmed: true })
  }

  return (
    <div className="topics-menu">
      <div className="topic-buttons">
        <div onClick={() => goToTopic('philosophy')}>Philosophy</div>
        <div onClick={() => goToTopic('technology')}>Technology</div>
        <div onClick={() => goToTopic('politics')}>Politics</div>
        <div onClick={() => goToTopic('music')}>Music</div>
        <div onClick={() => goToTopic('business')}>Business</div>
        <div onClick={() => goToTopic('literature')}>Literature</div>
        <div onClick={() => goToTopic('fitness')}>Fitness</div>
        <div onClick={() => goToTopic('photography')}>Photography</div>
        <div onClick={() => goToTopic('news')}>News</div>
      </div>
    </div>
  )
```